superset/views/core.py

```diff
@@ -2169,6 +2169,8 @@ def dashboard(self, dashboard_id):
             else:
                 dashboard_view = 'v1'
                 prompt_v2_conversion = not force_v1
+                if force_v2_edit:
+                    dash_edit_perm = False
```

Comment:
this line disallow owner to save original v1 dashboard. `Edit dashboard` button in Header controls used `dash_save_perm`, which is not affected.